posts/Davide/2018-05-25-hibernate-ogm-5-4-Beta1-released.adoc

```diff
@@ -0,0 +1,84 @@
+= "Hibernate OGM 5.4.0.Beta1 release"
+Davide D'Alto
+:awestruct-tags: [ "Hibernate OGM", "Releases" ]
+:awestruct-layout: blog-post
+---
+
+http://hibernate.org/ogm/releases/5.4/#get-it[Hibernate OGM 5.4.0.Beta1] has been released!
+
+The main differences from the previous release is the upgrade to Hibernate ORM 5.3.0.Final, 
+the support for Infinispan Remote native queries and the use of cluster counters
+for local caches when generating sequences with Infinispan Embedded.
+
+All the changes are described in detail in the
+https://hibernate.atlassian.net/secure/ReleaseNote.jspa?projectId=10160&version=31632[release notes].
+
+== Hibernate ORM 5.3
+
+You can find all the details about it in the
+http://in.relation.to/2018/05/14/hibernate-orm-530-final-release/[Hibernate ORM 5.3.0.Final release post].
+The main things I will mention here is that it comes with support for JPA 2.2 and many
+performance improvements.
+
+== Infinispan Remote queries
+
+You can execute queries in Infinispan using the string query language called
```

Comment:
> string query language

I would remove the "string" here, it looks weird.

Reply:
ok, the example should be enough